src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/ShowCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.logic.wanderlustlogic.wanderlustcommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.wanderlustlogic.wanderlustcommands.exceptions.CommandException;
+import seedu.address.model.travelplanner.Model;
+
+public class ShowCommand extends Command {
```

Comment:
Class is missing JavaDoc. I'm not sure if this is allowed since it passes CI?

Reply:
Resolved.